packages/dx-react-grid-material-ui/src/templates/table-detail-toggle-cell.jsx

```diff
@@ -5,41 +5,49 @@ import { withStyles } from 'material-ui/styles';
 
 import ChevronRight from 'material-ui-icons/ChevronRight';
 import ExpandMore from 'material-ui-icons/ExpandMore';
+import IconButton from 'material-ui/IconButton';
 
 const styles = theme => ({
   toggleCell: {
     textAlign: 'center',
-    cursor: 'pointer',
     textOverflow: 'initial',
+    paddingTop: '0px',
+    paddingBottom: '0px',
   },
   toggleCellIcon: {
     verticalAlign: 'middle',
     display: 'inline-block',
-    height: theme.spacing.unit * 3,
-    marginLeft: '-6px',
+    height: theme.spacing.unit * 6,
+    width: theme.spacing.unit * 6,
+    marginLeft: '-18px',
   },
 });
 
 const TableDetailToggleCellBase = ({
   style, expanded, classes, toggleExpanded,
-}) => (
-  <TableCell
-    className={classes.toggleCell}
-    style={style}
-    onClick={(e) => {
-      e.stopPropagation();
-      toggleExpanded();
-    }}
-  >
-    <span className={classes.toggleCellIcon}>
-      {
-        expanded
-          ? <ExpandMore />
-          : <ChevronRight />
-      }
-    </span>
-  </TableCell>
-);
+}) => {
+  const handleClick = () => {
+    toggleExpanded();
+  };
+  return (
+    <TableCell
+      className={classes.toggleCell}
+      style={style}
+    >
+      <IconButton
+        tabIndex={0} // eslint-disable-line jsx-a11y/no-noninteractive-tabindex
```

Comment:
Looks like there is no need to explicitly specify the `tabIndex` property for the button element. It can get focus by default.